packages/website/components/loading/loading.js

```diff
@@ -16,7 +16,7 @@ export const SpinnerColor = {
 };
 
 /**
- * @param {Object} [props]
```

Comment:
This was required for `next build` to build locally for me with tsc, and makes sense to clarify that the argument destructuring wont work if `props === undefined`